File: src/main/Java/boj_step2/boj2753.java
import java.io.*;

public class boj2753 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int year = Integer.parseInt(reader.readLine());
        int isLeap = ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) ? 1 : 0;
        writer.append(String.valueOf(isLeap));
        writer.close();
    }
}

File: src/main/kotlin/others/boj2753.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.time.Year

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val year = r.readLine()!!.toInt()
    w.write(((year % 4 == 0 && year % 100 != 0) || year % 400 == 0).compareTo(false).toString())
    w.close()
}